linera-service/tests/end_to_end_tests.rs

```diff
@@ -1500,6 +1503,9 @@ async fn test_wasm_end_to_end_matching_engine(config: impl LineraNetConfig) -> R
             &[token0.forget_abi(), token1.forget_abi()],
         )
         .await?;
+    node_service_admin.process_inbox(&chain_admin).await?;
+    node_service_a.process_inbox(&chain_a).await?;
+    node_service_b.process_inbox(&chain_b).await?;
```

Comment:
These ones look too early. We want to make sure that the RequestApplication messages are processed, then their answers.

Reply:
Ok, I changed the position of the `process_inbox` statements.